### PR TITLE
Feature/new home unit tests

### PR DESCRIPTION
**Pull request by nickguendling**
## Description
Add unit tests for the new Home Screen implementation

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-1958

### CODE REVIEW

File: src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme [hunk 1]
```diff
@@ -114,7 +114,6 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      language = "en"
```
Comment (marcussc) [inline src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme:117]: Was this a conscious Decision?

Comment (nickguendling) [reply src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme:117]: Yes. I don't think the App should start in English every time. Or do you know that somebody set this to English for everyone on purpose?

Comment (marcussc) [reply src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme:117]: Just checking, sometimes it happens to me that I change something in the scheme and forget to revert it :D

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme (hunk 1 above)
Comment (KaiTeuber) [inline src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme:117]: 👍

File: src/xcode/ENA/ENA/Source/Scenes/Home/__tests__/HomeTableViewModelTests.swift [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+class HomeTableViewModelTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/Home/__tests__/HomeTableViewModelTests.swift:16]: remove not needed code

**Review by KaiTeuber — APPROVED** [review]

**Pull request merged**